test/new-e2e/pkg/utils/e2e/client/host.go

```diff
@@ -516,11 +517,23 @@ func buildCommandOnWindows(h *Host, command string, envVar EnvVar) string {
 
 		cmd += fmt.Sprintf("$env:%s='%s'; ", envName, envValue)
 	}
-	cmd += fmt.Sprintf("%s; ", command)
-
-	for envName := range envVar {
-		cmd += fmt.Sprintf("$env:%s='%s'; ", envName, envVarSave[envName])
-	}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Was this removed on purpose? we were adding them to reset the env vars to previous values after command execution, is it not required on PowerShell?

Reply:
yes, it was the cause of the bug. See PR description and code comment for details.